libsplinter/src/protocol/mod.rs

```diff
@@ -21,7 +21,7 @@ pub mod service;
 
 // Peer authorization protocol versions
 #[cfg(any(feature = "trust-authorization", feature = "challenge-authorization"))]
-pub const PEER_AUTHORIZATION_PROTOCOL_VERSION: u32 = 1;
+pub const PEER_SPLINTER_PROTOCOL_VERSION: u32 = 1;
```

Comment:
This changes should be split into a different commit.

Reply:
This was unintentional. Removed from the commit.